.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "third_party/hiredis"]
 	path = third_party/hiredis
 	url = https://github.com/redis/hiredis.git
+[submodule "third_party/mosquitto"]
```

Comment:
With such huge feature, I recommended make import library another PR. And how long for mqtt library to compile btw?

Reply:
30 seconds is actually a lot though. I mean how long is `eclipse/mosquitto` particularly.

Reply:
that 30 seconds it the time spent by compiling `eclipse/mosquitto` only.

Reply:
can you also test this on node1. I'm concerned about CI performance.

Reply:
I will provide the elapsing time running on node1 hrer later

Reply:
It is about 17 seconds on node1